lambda_function.py

```diff
@@ -44,27 +44,29 @@ def lambda_handler(event, lambda_context):
                 and (k in message["user"] for k in ("values"))
                 and (k in message["purpose"]["params"] for k in ("platform", "id"))
             ):
+                # sets is_multiple_entry
+                # if the length of user input list has more than 1 userID
+                if len(message["user"]["values"]) > 1:
+                    row["is_multiple_entry"] = True
                 # parsing through each user ID and its respective valid flag
                 for each in message["user"]["values"]:
                     # the key values are the column names in the BQ table.
-                    row["timestamp"] = message["dateTime"]
+                    row["attendance_timestamp"] = message["dateTime"]
                     row["purpose_type"] = message["purpose"]["type"]
-                    row["purpose_sub_type"] = message["purpose"]["subType"]
+                    row["purpose_subtype"] = message["purpose"]["subType"]
                     row["platform"] = message["purpose"]["params"]["platform"]
                     row["platform_id"] = message["purpose"]["params"]["id"]
                     row["auth_type"] = message["authType"]
                     row["user_id"] = each["userID"]
                     row["user_data_validated"] = each["valid"]
-
+                    print(row)
```

Comment:
Is this for logging?

Reply:
actually not really logging, just for testing. I removed it.